src/text.rs

```diff
@@ -13,17 +13,21 @@ pub fn render_text(scene: &mut Scene, transform: Affine, layout: &Layout<Brush>)
             let run = glyph_run.run();
             let font = run.font();
             let font_size = run.font_size();
-            let font = vello::peniko::Font::new(font.data().0.clone(), font.index());
             let style = glyph_run.style();
+            let synthesis = run.synthesis();
+            let glyph_xform = synthesis
+                .skew()
+                .map(|angle| Affine::skew(angle.to_radians().tan() as f64, 0.0));
```

Comment:
Is this doing "fake italics"?

Reply:
Yeah, this always existed in parley but I never hooked it up in xilem. Dropping fount (and swash as a font parser for the fallback engine) required a new synthesis type and some conversions so I decided to add the integration to xilem to make sure it all still works.

Reply:
Awesome. I don't suppose "fake bold" would be as easy? Not a blocker for me, but would definitely be nice to have.

Reply:
Almost as easy. The synthesis object notes when it should be applied but we don’t yet have code in vello to dilate the outlines. Ideally we’d like to do this on the GPU at some point but we can and should probably hack it in on the CPU side for completeness until that is done.